--- a/setup.py
+++ b/setup.py
@@ -18,7 +18,7 @@
     platform_string = ":sys_platform=='%s'" % p
     extras_require[platform_string] = ["psutil"]
     if p in ("linux", "linux2"):
-        extras_require[platform_string].append("ld")
+        extras_require[platform_string].append("distro")
 
 # Get e3 version from the VERSION file.
 version_file = os.path.join(os.path.dirname(__file__), "VERSION")
--- a/src/e3/os/platform.py
+++ b/src/e3/os/platform.py
@@ -78,12 +78,12 @@
 
         # Fetch linux distribution info on linux OS
         if cls.uname.system == "Linux":  # linux-only
-            import ld
+            import distro
 
             cls.ld_info = {
-                "name": ld.name(),
-                "major_version": ld.major_version(),
-                "version": ld.version(),
+                "name": distro.name(),
+                "major_version": distro.major_version(),
+                "version": distro.version(),
             }
 
         # Fetch core numbers. Note that the methods does not work
